### DISCUSSION TITLE
请教 Hexo update time 用法

### DISCUSSION THREAD

**Discussion by mobeicanyue** [hexojs/hexo#5425] [Question / Answer] [answered]
由于 Hexo md文件在上传到 Github Action 构建的时候是新文件，所以更新时间都是显示的 Action 的构建时间。我的解决办法是 写一个脚本遍历目录下的 md 文件并将其修改时间更改为文件 commit 的时间。

这并不是长久之计，因为我有可能会更换系统或者迁移博客，所以记录下 update time 是更好的选择，但是手动填写来维护 update time 又过于不方便。

我近来阅读了 hexo 的文档，发现了这个，想请教一下这个参数的用法：
![image](https://github.com/hexojs/hexo/assets/81098819/7fc07065-8268-4129-bbc5-667a91d227f6)

是不是填写了 updated_option: date。这个参数的话，在 hexo g 的时候，hexo 会更新 md 文件表头的时间？还是怎么样

**Comment by uiolee** [comment, marked as answer]
What it means is that when there is no `updated` value in front-matter, the `date` value will be regarded as the `updated` value

**Reply by mobeicanyue** [reply]
It's only take effect on local machine?
Does it take effect on remote machine? Because it has not persistent storage. Remote file update time is action run time

**Reply by uiolee** [reply]
I can't understand what you mean.
it always takes effect. The `updated`, `date` values ​​are stored in [front-matter](https://hexo.io/docs/front-matter)

**Reply by mobeicanyue** [reply]
Yes, but when I modify the article, does the `updated` value could be update by hexo? Or I need to Manual modification?

**Reply by uiolee** [reply]
No. You need specify it manually.

**Reply by mobeicanyue** [reply]
So, It will only takes effct at create time? Do you have any idea about maintain it conviniently?

**Reply by uiolee** [reply]
no

**Reply by mobeicanyue** [reply]
Oh, thanks for your answers. God bless you.

**Comment by lucmann** [comment]
@mobeicanyue 请问现在有更好的解决方法了吗？
